examples/js/effects/VREffect.js

```diff
@@ -9,7 +9,7 @@
  *
  */
 
-THREE.VREffect = function ( renderer, onError ) {
+THREE.VREffect = function( renderer, onError ) {
```

Comment:
@zz85 Is it possible to make https://zz85.github.io/mrdoobapproves/ add a space in anonymous functions?

```js
function ( par1, par2 ) {

	...

}
```